src/Sulu/Bundle/SnippetBundle/Resources/public/js/components/snippet-content/main.js

```diff
@@ -275,6 +293,24 @@ define([], function() {
             }
 
             this.setData(data, false);
+        },
+
+        /**
+         * Removes references linking to deleted snippets.
+         *
+         * @param existingSnippets {object} The data returned from the api containing all existing snippets
+         */
+        removeInvalidReferences: function(existingSnippets) {
```

Comment:
Can you please move the method outside of this object? Right next to the `isArrayContainingSnippet` method? Then it won't be created for every single instance of this component.